subprojects/test-kit/src/integTest/groovy/org/gradle/testkit/runner/GradleRunnerUnsupportedGradleVersionFailureIntegrationTest.groovy

```diff
@@ -33,7 +36,7 @@ class GradleRunnerUnsupportedGradleVersionFailureIntegrationTest extends BaseGra
         result.output
 
         then:
-        def e = thrown UnsupportedFeatureException
-        e.message == "The version of Gradle you are using (1.1) is not supported by TestKit. TestKit supports all Gradle versions 1.2 and later."
+        def e = thrown UnsupportedVersionException
```

Comment:
❌ Why did this test change?

Reply:
I was trying to make all unsupported message unified. Now it's reverted.